docs/Chapter02-Overview-LoggingIn-and-Authentication.md

```diff
@@ -180,49 +177,49 @@ Some services allow you to choose different methods for your second factor. We
 
 ###  More about 2FA
 
-Enabling two-factor authentication is one of the best security practices
-that you can do and there’s lots more to read up about it if you want.
+Enabling two-factor authentication is one of the best security practices that you can do and there’s lots more to read up about it if you want.
 
--   [*OpenNews article on
-    2FA*](https://source.opennews.org/articles/two-factor-authentication-newsrooms/)
--   [*TwoFactorAuth.org*](https://twofactorauth.org/), a place to see
-    what services offer two-factor authentication and what kind
--   [*EFF’s guide to enabling 2FA on common
-    services*](https://www.eff.org/deeplinks/2016/12/12-days-2fa-how-enable-two-factor-authentication-your-online-accounts)
--   [*TurnOn1FA.com*](https://www.turnon2fa.com/) has lots of specific
-    tutorials on how to turn on 2FA for many services.
--   [*OpenNews security training chapter on
-    2FA*](Chapter02-07-Two-Factor-Authentication.html)
+-   [*Freedom of the Press Foundation: "Two-factor authentication for beginners*](https://freedom.press/training/2fa-beginners/)
+-   [*2fa.directory*](https://2fa.directory/), a place to see what services offer two-factor authentication, and what kind they offer
+-   [*OpenNews security training chapter on 2FA*](https://securitytraining.opennews.org/en/latest/Chapter02-07-Two-Factor-Authentication.html)
 
 ##  Biometric authentication
 
 Often used in place of a password, biometric authentication is using something that is unique to your physical body to prove that you are you. The most common is fingerprint scanning on our phones, as well as the face recognition that Apple has recently introduced.
 
 Biometric authentication has its pros and cons (need reference). If you want to use biometric authentication, it is suggested to enhance it with a password or PIN; for instance, on Android, you can require the password every few days. If you are at a higher risk than most, such as working on very sensitive materials, use both a password and biometrics every time.
 
-The use of biometric authentication is not new, but its consumer use is relatively new, and it really isn’t known what the [*practical*](https://views-voices.oxfam.org.uk/aid/2017/11/biometrics-help-us-answer-question) or [*legal*](https://www.popsci.com/iphonex-facial-recognition-legal-unlock) consequences of it are.
+The use of biometric authentication is not new, but its consumer use is relatively new, we are still learning about the [*practical*](https://techcrunch.com/2021/07/26/court-orders-us-capitol-rioter-to-unlock-his-laptop-with-his-face/) and [*legal*](http://www.pennstatelawreview.org/print-issues/face-it-the-convenience-of-a-biometric-password-may-mean-forfeiting-your-fifth-amendment-rights/) implications of biometric device unlock software. In general, in risky situations where there is a reasonable chance your device might be seized (e.g., when visiting a border or protest) it may be best to disable biometrics.
```

Comment:
I think you can make a stronger recommendation here - face/fingerprint unlocking should be disabled in situations where arrest/search and seizure are more likely.

Reply:
Tried to strengthen the recommendation here.